src/org/zaproxy/zap/extension/pscanrules/XFrameOptionScanner.java

```diff
@@ -54,6 +59,9 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			case DEFAULT: 
 			case LOW: 		
 			case OFF: } 
+		
+		String metaXFO = getMetaXFOEvidence(source);

```

Comment:
This statement could be moved to inside the next if statement, around line 86 (it's not needed/used outside).